src/ocean/core/array/Transformation.d

```diff
@@ -666,3 +666,50 @@ unittest
     int x;
     int[] arr = toArray(x);
 }
+
+/*******************************************************************************
+
+    Assigns elements to an array.
+
+    The function does not allocate GC memory and ensures the length of the
+    array matches the number of variadic arguments.
+
+    Params:
+        T = the type of array element
+        array = the array to assign the elements to
+        elements = the elements to add to the array
+
+*******************************************************************************/
+
+public void assignToArray (T) (T[] array, T[] elements...)
+{
+    array[] = elements[];
+}
+
+///
+unittest
+{
+    int[10] array;
+    testNoAlloc(array.assignToArray(9, 8, 7, 6, 5, 4, 3, 2, 1, 0));
```

Comment:
perhaps add as well:

```
auto before_ptr = array.ptr;
testNoAlloc(array.assignToArray(...));
test!("is")(before_ptr, array.ptr); 
```

Reply:
no problem, I'll update it as simplify a bit more the unit tests.